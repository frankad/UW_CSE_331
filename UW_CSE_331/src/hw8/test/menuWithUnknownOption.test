# Verify that when unknown option of a menu
r r
# Verify again unknown option
abc

q