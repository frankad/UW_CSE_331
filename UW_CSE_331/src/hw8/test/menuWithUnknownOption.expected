Menu:
	r to find a route
	b to see a list of all buildings
	q to quit
	
Enter an option ('m' to see the menu): # Verify that when unknown option of a menu
Unknown option

Enter an option ('m' to see the menu): # Verify again unknown option
Unknown option

Enter an option ('m' to see the menu):